pkg/controller/sub_controller/disaggregated_cluster/computegroups/util.go

```diff
@@ -17,8 +17,36 @@
 
 package computegroups
 
+import (
+	dv1 "github.com/apache/doris-operator/api/disaggregated/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
 // regex
 var (
 	compute_group_name_regex = "[a-zA-Z](_?[0-9a-zA-Z])*"
 	compute_group_id_regex   = "[a-zA-Z](_?[0-9a-zA-Z])*"
 )
+
+func ownerReference2ddc(obj client.Object, cluster *dv1.DorisDisaggregatedCluster) bool {
+	if obj == nil {
+		return false
+	}
+
+	ors := obj.GetOwnerReferences()
+	for _, or := range ors {
+		if or.Name == cluster.Name && or.UID == cluster.UID {
```

Comment:
Should type judgment be added?     or.Kind == "DorisDisaggregatedCluster"

Reply:
UID is the  unique identifier of a resource, so only using the UID is ok, use the name for double-check and for code reading.